Trustroots/trustroots
# fix podman, update some stuff, remove redundant stuff

#### Proposed Changes

* Add volume instruction that makes it work with rootless containers
* Remove redundancy in environment variable declaration
* Remove unnecessary fluff from docker-compose.yml

#### Testing Instructions

* Install docker if you don't have it yet: https://docs.docker.com/engine/install/
* Do the config/env/local.js thing described here: https://team.trustroots.org/Install-Docker.html
* docker-compose up
* should work (localhost:3000)

Fixes #2581
